### PR TITLE
added translations for subscription status

### LINKED ISSUE #1355: Status of account history items untranslated
Status is currently directly shown as given by API call from backend - should be rather translated using resources:

Bildschirmfoto 2021-12-13 um 12 53 26

### PR DESCRIPTION
**Pull request by norbertschuler**
Fix #1355

Changes in this pull request:
- added translations for subscription status taken from backend at https://github.com/Plant-for-the-Planet-org/treecounter-platform/blob/374b26f1658ce20a020254aaf1941f525b62e312/src/Entity/PaymentRecurrency.php#L64-L72
- replaced non existing status `pause` with `paused` fixing sorting

### CODE REVIEW

**Comment by sagararyal** [comment]
#### 📝 Changed routes:


| [**/profile/recurrency**](https://planet-webapp-quvfjdsyp-planetapp.vercel.app/profile/recurrency) |
|:-:|
|<a href="https://planet-webapp-quvfjdsyp-planetapp.vercel.app/profile/recurrency">Screenshot of /profile/recurrency</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/z5bQUttI9qoiybI0P4HvqgePDvoTfkFTrcKhKHewHpUeKlaSBSOzZsfaPA-yblhQhqJp3wEOiA3feoI_k8zabg.png">(view full size)</a> |







Commit ec96851bfd8b53aec33bd03d0ae2e62233a98df2 (https://planet-webapp-quvfjdsyp-planetapp.vercel.app).

File: public/static/locales/en/me.json [hunk 1]
```diff
@@ -194,5 +194,13 @@
...
-  }
+  },
+  "active": "active",
+  "canceled": "canceled",
+  "incomplete": "incomplete",
+  "incomplete_expired": "incomplete & expired",
+  "past_due": "past due",
+  "unpaid": "unpaid",
+  "trialing": "trialing",
+  "paused": "paused"
```
Comment (sagararyal) [inline public/static/locales/en/me.json:205]: For the user, we only show
Active --> active/trialing
Pending --> Incomplete, 
Cancelled --> incomplete/expired/canceled
Paused -> paused

Comment (norbertschuler) [reply public/static/locales/en/me.json:205]: Ok. That could easily be done by changing the translations.

Comment (sarveshpro) [reply public/static/locales/en/me.json:205]: So I am assuming this PR is good to merge

**Review by sarveshpro — APPROVED** [review]

File: public/static/locales/en/me.json (hunk 1 above)
Comment (norbertschuler) [reply public/static/locales/en/me.json:205]: I will change the resources in this PR as following (so we do not have to wait for a translation run):

```
  "active": "Active",
  "canceled": "Cancelled",
  "incomplete": "Pending",
  "incomplete_expired": "Cancelled",
  "past_due": "Past due",
  "unpaid": "Unpaid",
  "trialing": "Active",
  "paused": "Paused"
```

As @sagararyal  did not say anything about unpaid or past_due I keep them.

**Pull request merged**